Switch to Better Questing Unofficial and Various Quest Book Fixes

Exaxxion opened the pull request:
Fixes #880 by adding Awakened Draconium and Tritanium dependencies directly to the quest.

Unfortunately this adds some lines across the page which might need some layout adjustments to address. It's not too bad though, currently it looks like this: 
![image](https://user-images.githubusercontent.com/11907282/155451296-97c5fa26-5217-40a4-b92e-7ac4a2843351.png)

Exaxxion committed:
Add missing dependencies to T8 Micro Miner quest

- Awakened Draconium
- Tritanium

ghzdude commented:
Could probably add an invisible Tritanium quest (connected to the actual Tritanium quest) to hide the errant line.

Exaxxion commented:
Might also make sense to make a ZPM Field Generator quest to redirect the lines more cleanly, but frankly I'm hoping that we can use the BQ fork sometime for stuff like CleanroomMC/BetterQuesting#24

Exaxxion committed:
Add GP Generation as prerequisite for Oil Drilling Rig

The XU2 Screen is needed which needs Stoneburnt.

Exaxxion committed:
Convert almost all fluid bucket item retrieval quests to fluid quests

Replaces all of the quests which exclusively request fluid in buckets to
instead simply request the fluid, allowing any suitable container to
qualify.

Quests with a mixture of items and fluid buckets are unchanged, to help
avoid confusion since mixed task types are on separate task pages within
the quest book, and it's not particularly obvious.

Only downside is that these quests might not sync automatically if a new
player joins the quest group after this change. Just pick up the fluids,
and it will complete normally.

Exaxxion commented:
Merged dev branch, but most notably converted almost every case of fluid buckets in quests to fluid retrieval tasks so any container qualifies. See https://github.com/Nomifactory/Nomifactory/pull/881/commits/528bb396cd2aa50af81b6edc4746fff2801a59cf for more details.

Exaxxion commented:
A few TODOs:
- [ ] Pattern terminal quest has wrong icon
- [ ] UV Circuits on Progression tab isn't linked to wetware
- [ ] Determine if we can control the ordering of prerequisites so they show up nicer in the "Requires" tooltip

Exaxxion committed:
Switch to Better Questing Unofficial

- Remove Better Questing 3.5.329 and replace with Better Questing
  Unofficial 1.0.0

- Remove Better Questing Standard Expansion 3.4.173 (no longer needed)

- Remove Bansoukou patch

Exaxxion commented:
Known Issue: BQ Quest Book mod is incompatible with the BQu version numbering restarting from 1.0.0, I have an open PR to switch it to 4.0.0 to avoid this problem at https://github.com/CleanroomMC/BetterQuesting/pull/43

Exaxxion commented:
Just fixing rebase conflicts; I didn't want the prerequisite ID sorting change in this PR as it would obfuscate the relevant changes.

Exaxxion committed:
Link UV Circuits progression tab quest to Wetware Arrays

Exaxxion committed:
Fix Pattern Terminal quest icon

Exaxxion committed:
Update Better Questing Unofficial to 4.0.0

- Fixes incompatibility with Better Questing Quest Book

Exaxxion committed:
Switch line from Tritanium to T8 to appear on Hover

Exaxxion committed:
Add one Chorus Flower as a reward for the Telepad quest

Fixes #641 without introducing balance problems

Exaxxion committed:
Add quest for Extended Processing Pattern Terminal

Exaxxion committed:
Fix wrong visibility setting on EPPT quest

Exaxxion committed:
Switch remaining fluid bucket tasks to fluid tasks

BQu displays them all on one tab so it won't be confusing like before.

Exaxxion committed:
Fix pluralization on a circuit quest, order prerequisites

Reordering in anticipation of CleanroomMC/BetterQuesting#45

Exaxxion committed:
Update BQu 4.0.0 to 4.0.1

Includes the prerequisite reordering PR

Exaxxion committed:
Switch line from Draconium to 2nd T7 Circuits to appear on hover

Exaxxion committed:
Update BQu 4.0.1 to 4.0.2
